tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -3510,10 +3502,10 @@ public function deleteGroup(string $group):void {
 	 * @throws GuzzleException
 	 */
 	public function groupHasBeenDeleted(string $group):void {
-		if (OcisHelper::isTestingWithGraphApi()) {
-			$this->graphContext->adminHasDeletedGroupUsingTheGraphApi($group);
-		} else {
+		if ($this->isTestingWithLdap()) {
 			$this->deleteGroup($group);
```

Comment:
Would it be possible to use this function directly here and remove the above `deleteGroup` function?
```suggestion
		if ($this->isTestingWithLdap()) {
			$this->deleteLdapGroup($group);
```

Reply:
should we did not need to check whether the group exists or not?

Reply:
you can add that step here or directly inside `deleteLdapGroup` function if possible